handbook/engineering/distribution/product/roadmap.md

```diff
@@ -0,0 +1,253 @@
+# Distribution product roadmap
+
+This living document is the product roadmap for the Distribution team.
+
+It is longer-term than our quarterly OKRs, and higher-level than our GitHub issues. Additionally, it documents dependencies of roadmap items and current owners.
+
+## Ordered & prioritized roadmap
+
+1. (Q1 2020) [Support customers in deploying Sourcegraph with 500k+ repositories](support-customers-in-deploying-sourcegraph-with-500k-repositories)
+1. (Q1 2020) [Kubernetes upgrades should have less merge conflicts](#kubernetes-upgrades-should-have-less-merge-conflicts)
+1. (Q1 2020) [Docker-compose is not released on time (on the 20th)](#docker-compose-is-not-released-on-time-on-the-20th)
+1. (Q1 2020) [Releasing Sourcegraph should be automated](#releasing-sourcegraph-should-be-automated)
+1. (Q1 2020) [Automatic e2e testing](#automatic-e2e-testing)
+1. (TBD) [Automatic Docker image testing](#automatic-docker-image-testing)
+1. (TBD) [Upgrades across multiple Sourcegraph versions should be easier](#upgrades-across-multiple-sourcegraph-versions-should-be-easier)
+1. (TBD) [Sourcegraph should be released daily](#sourcegraph-should-be-released-daily)
+1. (TBD) [All site admins should have alerting set up to be notified when Sourcegraph is unhealthy](#all-site-admins-should-have-alerting-set-up-to-be-notified-when-sourcegraph-is-unhealthy)
+1. (TBD) [Push site admins to use Docker Compose or Kubernetes for production deployments](#push-site-admins-to-use-docker-compose-or-kubernetes-for-production-deployments)
+1. (TBD) [Add monitoring for common critical issues](#add-monitoring-for-common-critical-issues)
+1. (TBD) [Monitoring federation](#monitoring-federation)
+1. (TBD) [GitOps for all internal infrastructure](#gitops-for-all-internal-infrastructure)
```

Comment:
^ @pecigonzalo this is an interesting list to re-visit today, cool to see how much progress we've made on all of these!